packages/itwinui-react/src/core/Select/Select.tsx

```diff
@@ -536,3 +540,197 @@ const MultipleSelectButton = <T,>({
     </>
   );
 };
+
+// ----------------------------------------------------------------------------
+
+const NativeSelect = React.forwardRef((props, forwardedRef) => {
+  const {
+    triggerProps,
+    options,
+    disabled,
+    placeholder,
+    defaultValue: defaultValueProp = placeholder !== undefined ? '' : undefined,
+    value: valueProp,
+    onChange: onChangeProp,
+    size,
+    status,
+    required,
+    ...rest
+  } = props;
+
+  return (
+    <InputWithIcon {...rest} ref={forwardedRef}>
+      <SelectButton
+        as='select'
+        size={size}
+        status={status}
+        disabled={disabled}
+        defaultValue={defaultValueProp}
+        value={valueProp === null ? '' : valueProp}
+        {...triggerProps}
+        onKeyDown={mergeEventHandlers(triggerProps?.onKeyDown, (event) => {
+          // Firefox does not open the menu on Enter, so we need to do it manually.
+          if (event.key === 'Enter') {
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
+            event.currentTarget.showPicker?.();
+          }
+        })}
+        onChange={mergeEventHandlers(triggerProps?.onChange, (event) => {
+          onChangeProp?.(event.currentTarget.value, event);
+        })}
+        required={required}
+      >
+        {placeholder !== undefined ? (
+          <option value='' disabled>
+            {placeholder}
+          </option>
+        ) : null}
+
+        {options.map((option) => (
+          <option key={option.value} {...option}>
+            {option.label}
+          </option>
+        ))}
+      </SelectButton>
+
+      <SelectEndIcon disabled={disabled} />
+    </InputWithIcon>
+  );
+}) as PolymorphicForwardRefComponent<'div', NativeSelectProps>;
+
+// This is a smaller, stricter subset of CustomSelectProps, because native
+// <select> only supports string values, and does not support icons or sublabels.
+type NativeSelectProps = SelectCommonProps & {
+  /**
+   * Selected option value.
+   *
+   * Must be a string, because it is passed as an attribute to the native <select>.
+   */
+  value?: string | null;
```

Comment:
It can be mentioned that passing `null` resets the value, similar to the value in `SelectMultipleTypeProps`.